### PR TITLE
ign -> gz Upstream Macro Migration : gz-msgs

### PR DESCRIPTION
**Pull request by methylDragon**
See: https://github.com/gazebo-tooling/release-tools/issues/737

### CODE REVIEW

**Review by chapulina — APPROVED** [review]
LGTM once upstream is merged.

File: src/ign.hh [hunk 1]
```diff
@@ -30,6 +30,7 @@ extern "C" GZ_MSGS_VISIBLE void cmdMsgList();
 
 /// \brief External hook to read the library version.
 /// \return C-string representing the version. Ex.: 0.1.2
-extern "C" GZ_MSGS_VISIBLE const char *ignitionMsgsVersion();
+extern "C" GZ_MSGS_VISIBLE const char *gzMsgsVersion();
+extern "C" GZ_MSGS_VISIBLE const char GZ_DEPRECATED(9) *ignitionMsgsVersion();
```
Comment (chapulina) [inline src/ign.hh:34]: Oh this header isn't installed (it isn't in the `include` directory), so we can do a hard tock

Comment (chapulina) [reply src/ign.hh]: [d67ca90](https://github.com/gazebosim/gz-msgs/pull/260/commits/d67ca90169437fe87af6876f6aa60d856596281c)

**Review by chapulina — APPROVED** [review]

**Comment by methylDragon** [comment]
@osrf-jenkins run tests please!

**Pull request merged**